src/com/hotmail/kalebmarc/textfighter/main/Game.java

```diff
@@ -193,16 +194,10 @@ public void start() {
 					Ui.println("2) Recovery Potion");
 					Ui.println("3) Back");
 					switch (Ui.getValidInt()) {
-						case 1:
-							Potion.use("survival");
-							break;
-						case 2:
-							Potion.use("recovery");
-							break;
-						case 3:
-							break;
-						default:
-							break;
+						case 1 -> Potion.use("survival");

```

Comment:
Might be worth adding a note in the README to use java 14+ as this isn't available in older versions... Up to you though, not a big deal, most people are probably not using a super old version of java

Reply:
The README kinda hints at using Java 11+ from one of the screenshots. I believe Java 8 is still the most common version. I reverted back to 11 and made changes as a result, tested working!